include/mbedtls/ecdh.h

```diff
@@ -280,7 +292,7 @@ int mbedtls_ecdh_make_public( mbedtls_ecdh_context *ctx, size_t *olen,
  * \brief       This function parses and processes a TLS ClientKeyExchange
  *              payload.
  *
- *              This is the second function used by a TLS server for ECDH(E)
+ *              This is the third function used by a TLS server for ECDH(E)
```

Comment:
Please add a reference to the first two functions or to a place where all the successive functions are listed.